docs/concepts/feature-generation.md

```diff
@@ -89,8 +107,32 @@ settings = MaterializationSettings("nycTaxiTable",
                                    backfill_time=backfill_time)
 ```
 
-This will generate features only for 2020/05/20 for me and it will be in folder:
-`abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/materialize_offline_test_data/df0/daily/2020/05/20`
+This will generate features from `2020/05/10` to `2020/05/20` and the output will have 10 folders, from
+`abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/materialize_offline_test_data/df0/daily/2020/05/10` to `abfss://feathrazuretest3fs@feathrazuretest3storage.dfs.core.windows.net/materialize_offline_test_data/df0/daily/2020/05/20`. Note that currently Feathr only supports materializing data in daily step (i.e. even if you specify an hourly step, the generated features in offline store will still be presented in a daily hierarchy).
```

Comment:
Maybe I'm wrong with the backend logic, but 2020/05/10 to 2020/05/20 should be 11 folders?

Reply:
Thanks... good catch!